packages/astro/src/events/error.ts

```diff
@@ -72,8 +72,6 @@ export function eventError({
 		cliCommand: cmd,
 		isFatal: isFatal,
 		anonymousMessageHint:
-			// https://github.com/typescript-eslint/typescript-eslint/issues/4820
-			// eslint-disable-next-line @typescript-eslint/prefer-optional-chain -- errorData may be false
```

Comment:
Explaining: I disable the rule in `.eslintrc.cjs`, so it doesn't need to be disabled here.